e2e/page-objects/loginPage.ts

```diff
@@ -0,0 +1,25 @@
+import { Page, expect } from '@playwright/test';
+import { HelperBase } from './helperBase';
+
+export class LoginPage extends HelperBase {
+  /**
+   * This method checks the Log in button visibility and then fills in email and password with special speed emulating real user typing
+   * @param email - should be email in form of string
+   * @param password - should be password in form of string
+   * @param speed - delay between keyboard clicks in ms. High numbers increase test execution!
+   */
+
+  async typeLoginPasswordAndLogin(email: string, password: string, speed: number) {
+    await expect(this.page.getByRole('button', { name: 'Log in' })).toBeVisible();
+    // Getting email field and type in like real user
+    const emailInput = this.page.getByRole('textbox').first();
```

Comment:
Can we locate email same way as password?

Reply:
the same way - not. some other ways - yes. 

<img width="548" alt="Monosnap Disaster Ninja 2024-04-18 18-34-44" src="https://github.com/konturio/disaster-ninja-fe/assets/112632451/0fe3cd84-50b8-4299-8310-cf06eade25e7">